javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,61 @@
+"use strict"
+console.log('Script loaded');
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector('section.products ul');
+console.log(productsUl);
+
+function renderProducts(products) {
+        
+         while (productsUl.hasChildNodes()) {
+        productsUl.removeChild(productsUl.firstChild);
+      }
+        
+        products.forEach((product)=>{const li = document.createElement('li');
+
+        let shipsToHTML = '';
+        product.shipsTo.forEach(country => shipsToHTML += `<li>${country}</li>`);
+
+        li.innerHTML = `
+            <ul>
+                <li>${product.name}</li>
+                <li>${product.price}</li>
+                <li>${product.rating}</li>
+                <ul class="ships-to">${shipsToHTML}</ul>
+            </ul>
+        `;
+        productsUl.appendChild(li);});
+}
+
+
+
+// Search products by inputting names
+
+
+document.getElementById("searched").addEventListener('input', (event)=>{
```

Comment:
```suggestion
document.querySelector("#searched").addEventListener('input', (event)=>{
```

Reply:
why should I use querySelector instead?

Reply:
You dont have to. But its good practice. The reason is that then using `getElementById` you need to give all elements you need to interact with an id. Id's should rarely be used in html. They are for very unique styles. 

querySelector is also consisten with css selectors so the selectors are the same no matter if you use css or js